layouts/shortcodes/config-file-existing-arrikto.html

```diff
@@ -0,0 +1 @@
+kfctl_existing_arrikto.0.7.0.yaml
```

Comment:
Is this used anywhere?

Reply:
Well spotted. No, it's not. It was used in a section, but we decided to move that section out of this doc. I've added a note to the corresponding issue: https://github.com/kubeflow/website/issues/1338